Source/JavaScriptCore/assembler/ARMv7Assembler.h

```diff
@@ -1320,6 +1323,13 @@ class ARMv7Assembler {
         m_formatter.twoWordOp12Reg4Reg4Reg4Imm8(static_cast<OpcodeID1>(opcode), rn, rt, rt2, offset);
     }
 
+    ALWAYS_INLINE void ldrex(RegisterID rt, RegisterID rn, ARMThumbImmediate imm)
```

Comment:
The immediate in ldrex is not the universal ARMThumbImmediate kind, so an int/unsigned would be more appropriate here + assertions that it's in the available range (multiples of 4 between 0 .. 1023 I believe but check the ARMARM)

Same for strex below.